test/collection/test_config.py

```diff
@@ -811,6 +811,40 @@ def test_config_with_vectorizer_and_properties(
             }
         },
     ),
+    (
+        Configure.Generative.databricks(),
+        {"generative-databricks": {}},
+    ),
+    (
+        Configure.Generative.databricks(
+            model="model",
+            base_url="https://api.databricks.com",
+            frequency_penalty=0.5,
```

Comment:
LLMs on databrick's don't have this parameter

Reply:
This (and the other params) were implemented because they're supported in the module as defined within Weaviate [here](https://github.com/weaviate/weaviate/blob/main/modules/generative-databricks/parameters/params.go). Are these values actually unused by the module so should be ignored by the our client-side implementation?

Reply:
Further, are these the only allowed params?
```go
const (
	temperatureProperty = "temperature"
	maxTokensProperty   = "maxTokens"

	topPProperty = "topP"
	topKProperty = "topK"
)
```

Reply:
yes, those are the only parameters that are implemented for now.

Reply:
Okay, have changed!